docs/getting-started/quickstart.md

```diff
@@ -61,110 +70,97 @@ Run the following command to deploy your app to DBOS Cloud:
 npx dbos-cloud app deploy
 ```
 
-This command first prompts you to login/sign up for DBOS Cloud.
-
-Then, it prompts you to provision a free [database instance](../cloud-tutorials/database-management.md) for your application.
-
-Finally, it uploads your code to DBOS Cloud and deploys your application.
-After about a minute, it should succeed and print `Successfully deployed <app-name>! Access your application at <URL>`.
+This command first prompts you to login, or register if this is your first time. Then, it prompts you to provision a database instance. Finally, it uploads your code to DBOS Cloud and deploys your app. In about a minute, it should succeed and print `Successfully deployed <app-name>! Access your application at <URL>`.
```

Comment:
It really is just 20 seconds now--don't think we need to say a minute.

Reply:
Should be all set now